repository_service_tuf_api/common_models.py

```diff
@@ -86,6 +86,36 @@ class TUFSigned(BaseModel):
 
     class Config:
         fields = {"type": "_type"}
+        # https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.extra
+        model_config = ConfigDict(
+            extra="allow",
+        )
+        extra = (
+            Extra.allow
+        )  # allow extra unrecognized fields (but it will be validated)
```

Comment:
Without context there seems to be a bit of a definition redundancy. Would you mind telling me, why you need
`Config.model_config.extra = "allow"` and `Config.extra = Extra.allow`? :)

Reply:
Sorry, my bad. Fixing.
It should have only `extra=Extra.allow`